runtime/queries/markdown_inline/highlights.scm

```diff
@@ -29,11 +27,8 @@
 ] @string.escape
 
 ; Conceal codeblock and text style markers
-([
-  (code_span_delimiter)
-  (emphasis_delimiter)
-] @punctuation.delimiter
-(#set! conceal ""))
+(code_span_delimiter) @markup.raw (#set! conceal "")
+(emphasis_delimiter) @markup.strong (#set! conceal "")
```

Comment:
Possible readability issue. They should be inside a grouping, similar to other queries with just `(#set!)`
```suggestion
((code_span_delimiter) @markup.raw 
  (#set! conceal ""))
((emphasis_delimiter) @markup.strong 
  (#set! conceal ""))
```